Intersect.Server.Core/Maps/MapInstance.cs

```diff
@@ -703,7 +704,8 @@ private void DespawnResources()
     /// <param name="x">The X location of this item.</param>
     /// <param name="y">The Y location of this item.</param>
     /// <param name="item">The <see cref="MapItem"/> to add to the map.</param>
-    private void AddItem(MapItem item)
+    /// <param name="source">The source of the item.</param>
```

Comment:
This should be first, and the `x` and `y` parameter documentation should be removed since they don't exist here anymore